Avoids duplicate symbol complaints surfaced by -fno-common.

The defined values are never used in the code.

$OpenBSD$

Index: uwm/workspaces.h
--- uwm/workspaces.h.orig
+++ uwm/workspaces.h
@@ -92,14 +92,6 @@ typedef struct {
 	long int	flags;
 } UDE_LWS_Layer;
 
-// These three lists will be declared in workspaces.c, and declared static
-// so that no other module has direct access to them
-NodeList Workspaces;
-NodeList Layers;
-NodeList WS_Layer_Defaults; // This will be the list which handles setting up
-							// defaults for where windows will be mapped to.
-// End three listings
-
 void UDE_LWS_InitializeWSL(void);
 void UDE_LWS_FinalizeWSL(void);
 UDE_LWS_Workspace *UDE_LWS_AllocWorkspace(void);
